pkg/destroy/aws/ec2helpers.go

```diff
@@ -786,7 +786,40 @@ func deleteEC2VPCPeeringConnection(ctx context.Context, client *ec2.EC2, id stri
 }
 
 func deleteEC2VPCEndpointService(ctx context.Context, client *ec2.EC2, id string, logger logrus.FieldLogger) error {
-	_, err := client.DeleteVpcEndpointServiceConfigurationsWithContext(ctx, &ec2.DeleteVpcEndpointServiceConfigurationsInput{
+	logger.Warn("Here 1234567")
+	output, err := client.DescribeVpcEndpointConnectionsWithContext(ctx, &ec2.DescribeVpcEndpointConnectionsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("service-id"),
+				Values: aws.StringSlice([]string{id}),
+			},
+		},
+	})
+
+	if err != nil {
+		logger.Warn("Unable to get the list of VPC endpoint connections connected to service: ", err)
+		logger.Warn("Attempting to delete the VPC Endpoint Service")
+	} else {
+		endpointList := make([]*string, len(output.VpcEndpointConnections))
+		for _, endpoint := range output.VpcEndpointConnections {
+			if aws.StringValue(endpoint.VpcEndpointState) != "rejected" {
+				endpointList = append(endpointList, endpoint.VpcEndpointId)
+			}
+		}
+
+		_, err = client.RejectVpcEndpointConnectionsWithContext(ctx, &ec2.RejectVpcEndpointConnectionsInput{
+			ServiceId:      &id,
+			VpcEndpointIds: endpointList,
+		})
+
+		if err != nil {
+			logger.Warn("Unable to reject VPC endpoint connections for service: ", err)
+			logger.Warn("Attempting to delete the VPC Endpoint Service")
+		}
+		logger.Info("Rejected")
```

Comment:
Almost all logs are just this one word statements like `deleted`, `disassociated` etc. Want me to change this too?

Reply:
The logger already has the `id` of the resource. If we were to change this, maybe it would be good to include `resourceType` in https://github.com/openshift/installer/blob/master/pkg/destroy/aws/ec2helpers.go#L95